lib/spaceship/tunes/tunes_client.rb

```diff
@@ -314,10 +314,16 @@ def app_version(app_id, is_live)
 
       # We only support platforms that exist ATM
       platform = platforms.find do |p|
-        ['ios', 'osx'].include? p['platformString']
+        ['ios', 'osx', 'appletvos'].include? p['platformString']
       end
 
-      raise "Could not find platform ios or osx for app #{app_id}" unless platform
+      raise "Could not find platform ios, osx or appletvos for app #{app_id}" unless platform
+
+      # If your app has versions for both iOS and tvOS we will default to returning the iOS version for now.
+      # This is intentional as we need to do more work to support apps that have hybrid versions.
+      if platforms.length > 1
+        platform = platforms.find { |p| p['platformString'] == "ios" }
```

Comment:
@KrauseFx does this look a bit better? Looks like hound isn't happy with find though.